packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/create/xcpd-response.ts

```diff
@@ -16,79 +16,77 @@ function createQueryByParameter(request: InboundPatientDiscoveryReq): object {
   const { id, samlAttributes, patientResource } = request;
 
   const queryByParameter = {
-    queryByParameter: {
```

Comment:
theres only one tiny change in this PR, which is deleting the redundant queryByParameter object thats inside the queryByParameter const